spec/test_app/app/controllers/basechurch/v1/bulletins_controller_spec.rb

```diff
@@ -132,7 +153,7 @@
 
         context "where published_at is not iso8601 compliant" do
           before do
-            invalid_attributes[:bulletins][:publishedAt] = 'sdafasdfdsa'
+            invalid_attributes[:data][:attributes][:publishedAt] = 'sdafasdfdsa'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.